app/templates/components/explorer/download-data-dropdown.hbs

```diff
@@ -0,0 +1,49 @@
+{{!-- <a class='button tiny gray text-orange text-weight-normal' {{action 'exportTableToCSV'}}>{{fa-icon 'download'}} Download Data <small>(CSV)</small></a> --}}
+
+<button
+  class="topic-select-toggle button gray expanded no-margin"
+  style="padding-right: 0;"
+  role="button"
+  {{on "click" this.toggleOpen}}
+>
+  <div class="grid-x full">
+    <div class="cell small-11">
+      Download Selected Tables
+      <span class="orange-bar-notification-count">
+        {{this.numSelected}}
+      </span>
+    </div>
+  </div>
+</button>
+
+{{#if this.open}}
+  <div class="download-data-dropdown">
+    <div class="download-data-dropdown-head">
+      Data Download Summary
+    </div>
+    <div class="download-data-dropdown-body">
+      <p class="head">Data Source</p>
+      <p>{{this.source.label}}</p>
+      <p class="head">Topics</p>
+      {{#if (eq this.source.label 'ACS')}}
```

Comment:
just a heads up I'll be changing the `source` objects so that you can reference `this.source.type`.
So here you can write `{{#if (eq this.source. type 'acs')}}`